llama-index-core/tests/workflow/examples/rag.py

```diff
@@ -95,10 +97,10 @@ async def synthesize(
             self.query = ev.get("query", "")
             return None
         elif isinstance(ev, QueryResult):
-            llm = OpenAI(model="gpt-3.5-turbo")
-            summarizer = Refine(llm=llm, verbose=True)
+            llm = Ollama(model="llama3.1:latest", request_timeout=120)
```

Comment:
Ollama is slow enough to make the streaming noticeable lol

Reply:
Should we explicitly pin 8B to let it run locally?